src/views/Alteration.vue

```diff
@@ -19,9 +19,6 @@
 
       <articles class="mt-10" />
 
-<!--      Commented out until required-->
-<!--      <agreement-type class="mt-10" />-->
-<!--      <detail class="mt-10" />-->
```

Comment:
Should I remove this from my PR as well?

Reply:
Shouldn't need too, i don't think* it could cause a conflict.